utils/src/main/java/com/cloud/utils/crypt/EncryptionSecretKeyChecker.java

```diff
@@ -55,8 +57,8 @@ public void init() {
         DbProperties.getDbProperties();
     }
 
-    public void check(Properties dbProps) throws IOException {
-        String encryptionType = dbProps.getProperty("db.cloud.encryption.type");
+    public void check(Properties dbProps, String property) throws IOException {
```

Comment:
```suggestion
    public void check(Properties properties, String property) throws IOException {
```

_props_  or _properties_ whatever